fdbclient/ClientLibManagement.actor.cpp

```diff
@@ -46,7 +47,7 @@ struct ClientLibBinaryInfo {
 #define ASSERT_INDEX_IN_RANGE(idx, arr) ASSERT(idx >= 0 && idx < sizeof(arr) / sizeof(arr[0]))
 
 const std::string& getStatusName(ClientLibStatus status) {
-	static const std::string statusNames[] = { "disabled", "available", "uploading" };
+	static const std::string statusNames[] = { "disabled", "available", "uploading", "download", "active" };
```

Comment:
What does the available state represent? Would this be for clients to opt in to download it?

Reply:
Available means it is possible to download the library, but the clients are not instructed to download it yet. Maybe we don't need the that state. I introduced it before designing the client-side behavior.